code/modules/atmospherics/machinery/portable/canister.dm

```diff
@@ -36,10 +36,14 @@
 	var/shielding_powered = FALSE
 	///The powercell used to enable shielding
 	var/obj/item/stock_parts/cell/internal_cell
-	///Is the cell hatch opened
-	var/cell_container_opened = FALSE
 	///used while processing to update appearance only when its pressure state changes
 	var/current_pressure_state
+	/// An assembly attached to the canister
+	var/obj/item/assembly_holder/rig = null
+	/// The person who attached an assembly to this canister, usually for ignition purposes in tandem with valve wire pulse
+	var/last_rigger = ""
```

Comment:
```suggestion
	var/mob/last_rigger
```

Also this var isn't being used correctly. The `usr` is being logged in every instance, which may as well be a rando walking into a proximity sensor rig and unknowingly set off a hellburn canister.